src/test/regress/expected/multi_extension.out

```diff
@@ -390,6 +390,7 @@ SELECT * FROM print_extension_changes();
 
 -- Test downgrade to 9.3-2 from 9.4-1
 ALTER EXTENSION citus UPDATE TO '9.4-1';
+ALTER EXTENSION citus UPDATE TO '9.5-1';
```

Comment:
It appears that you did not update test outputs after introducing extra steps for downgrades